apps/src/lib/util/mlApi.js

```diff
@@ -0,0 +1,32 @@
+/* global Promise */
+
+import {predict} from '@cdo/apps/MLTrainers';
+
+export const commands = {
+  async getPrediction(opts) {
+    return new Promise((resolve, reject) => {
+      $.ajax({
```

Comment:
we should import jquery into this file to make sure it's available when this method is called

Reply:
in the long term, we will need to request this data once and store it in memory because right now this will make 2 network requests every time the block is executed